fri/src/options.rs

```diff
@@ -12,7 +12,7 @@ use math::StarkField;
 #[derive(Clone, PartialEq, Eq)]
 pub struct FriOptions {
     folding_factor: usize,
-    max_remainder_size: usize,
+    remainder_size: usize,
```

Comment:
Similar comment to the one before: should we rename this into `max_remainder_degree`?

Reply:
I have renamed all related variables as `max_remainder_degree_plus_1`. It is a bit verbose but accurate.